src/components/PeriodDimension/FixedPeriodSelect.js

```diff
@@ -1,5 +1,5 @@
-import propTypes from '@dhis2/prop-types'
 import { SingleSelectField, SingleSelectOption } from '@dhis2/ui'
+import propTypes from 'prop-types'
```

Comment:
Typo?
```suggestion
import PropTypes from 'prop-types'
```

and also rename in `propTypes` below.